recipe/meta.yaml

```
@@ -1,73 +1,56 @@
 {% set name = "onnxruntime" %}
-{% set version = "1.12.1" %}
+{% set version = "1.15.1" %}
 
 package:
   name: {{ name|lower }}{{ suffix }}
   version: {{ version }}
 
 source:
   - url: https://github.com/microsoft/onnxruntime/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: 302e5a0f368c7d048a9acd1227ac226148ed9c944f8b67d1077ca1b3bb3dcc5b
+    sha256: 93a9b6f148639938ccbaa48d0f641d8f33312fdfcc69ee9466e11362b43917c4
     patches:
-      - 0001_dont_vectorize.patch
-      - 0002_use_system_nsync.patch
-      - 0003_dont_download_deps.patch
-  - url: https://github.com/onnx/onnx/archive/f7ee1ac60d06abe8e26c9b6bbe1e3db5286b614b.tar.gz
-    sha256: 0624860bc300fada6c0f2b5fd18043528bd0e149aad9b3c329d478ab185f17d3
-    folder: onnx
-  - url: https://gitlab.com/libeigen/eigen/-/archive/d10b27fe37736d2944630ecd7557cefa95cf87c9/eigen-d10b27fe37736d2944630ecd7557cefa95cf87c9.tar.gz
-    sha256: a3c10a8c14f55e9f09f98b0a0ac6874c21bda91f65b7469d9b1f6925990e867b
-    folder: eigen
-  - url: https://github.com/pytorch/cpuinfo/archive/5916273f79a21551890fd3d56fc5375a78d1598d.zip
-    sha256: 2a160c527d3c58085ce260f34f9e2b161adc009b34186a2baf24e74376e89e6d
-    folder: pytorch_cpuinfo
-  - url: https://github.com/nlohmann/json/archive/refs/tags/v3.10.5.tar.gz
-    sha256: 5daca6ca216495edf89d167f808d1d03c4a4d929cef7da5e10f135ae1540c7e4
-    folder: json
-  - url: https://github.com/microsoft/wil/archive/e8c599bca6c56c44b6730ad93f6abbc9ecd60fc1.tar.gz  # [win]
-    sha256: 388156309853aaa6e01d177f9b68bd2c6943e5fc073ac6786f9383c1b4438812 # [win]
-    folder: wil # [win]
+      - never_call_install_python_deps.patch
+    #   Workaround for https://github.com/conda-forge/onnxruntime-feedstock/pull/56#issuecomment-1586080419
+      - windows_workaround_conflict_onnxruntime_dll_system32.patch  # [win]
+      - 16337.patch
+    #   Workaround for https://github.com/microsoft/onnxruntime/issues/13225
 
 build:
   number: 0
-  skip: True # [s390x or ppc64le]
+  skip: True # [py>310 or ppc64le or s390x]
   ignore_run_exports_from:
-    - libprotobuf
+    - zlib
+  entry_points:
+    - onnxruntime_test = onnxruntime.tools.onnxruntime_test:main
+  missing_dso_whitelist:     # [win64]
+    - '**/api-ms-win-core-path-l1-1-0.dll'     # [win64]
 
 requirements:
   build:
+    - python
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
     - git
```

```suggestion
    - git  # [not win]
```
**Prefect** already has git on `win64`. Also, this change will prevent crashing the `win64` agent on Prefect. See a corresponding error in the anaconda-linter log message